include/d/actor/d_a_obj_mkiek.h

```diff
@@ -2,12 +2,20 @@
 #define D_A_OBJ_MKIEK_H
 
 #include "d/d_bg_s_movebg_actor.h"
+#include "m_Do/m_Do_ext.h"
 
 namespace daObjMkiek {
+
+    enum daObjMkiek_State {
+        STATE_0 = 0,
+        STATE_1 = 1,
+        STATE_2 = 2,
+    };
+
     class Act_c : public dBgS_MoveBgActor {
     public:
-        void prm_get_sound() const {}
-        void prm_get_swSave() const {}
+        int prm_get_sound() const;
+        int prm_get_swSave() const;
```

Comment:
What was the reason for defining these inlines in the .cpp instead of the .h? Is that required for matching?